### PR TITLE
Fix for machinelearningmastery.com

### PR DESCRIPTION
**Pull request by hjpoe**
Fixes the dark background of an image in Dark mode. I did INVERT instead of CSS because the CSS I tried made it have a black background in Light mode.

### CODE REVIEW

**Comment by hjpoe** [comment]
I guess I should have put a [link to the page](https://machinelearningmastery.com/object-recognition-with-deep-learning/) it's on.

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -7345,7 +7345,7 @@ img[src*="/img/"]
 
 CSS
 .star-on-png::before {
-    color: #FCAD03 !important; 
+    color: #F$2.34 !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:7348]:
```suggestion
    color: #FCAD03 !important; 
```

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:7348]: Please revert this change.

**Review by Gusted — APPROVED** [review]
LGTM

**Pull request merged**